addons/pos_self_order/static/src/mobile/pages/order_cart/order_cart.xml

```diff
@@ -13,7 +13,7 @@
             <div class="d-flex align-items-center w-100 p-3 bg-white shadow-lg">
                 <div t-attf-class="{{ state.cancelConfirmation ? 'flex-row-reverse' : '' }} d-flex mt-2 flex-fill">
                     <button
-                        t-if="order.hasNotAllLinesSent() and order.lines.length > 0"
+                        t-if="!order.isSavedOnServer and order.lines.length > 0"
```

Comment:
it's often hard to see at first glance that this is a getter. It would be nice to just use normal methods.